filename.go

```diff
@@ -0,0 +1,10 @@
+package slinguist
+
+func GetLanguageByFilename(filename string) (lang string, safe bool) {
+	lang, safe = languagesByFilename[filename]
+	if lang == "" {
+		lang = OtherLanguage
```

Comment:
Can we make OtherLanguage the zero value of a lang type to get rid of the extra code to handle OtherLanguage?